sim/validator.py

```diff
@@ -178,17 +98,11 @@ def performance_test(results_path, btb_seq_path, reference_path, exist_ok=False,
     if branch:
         checkout(btb_seq_backup_path, branch)
```

Comment:
Should we cleanup by checking out master at the end? Or maybe check which branch is checked out before running the code and "re-checkout" that branch at the end? Just a TODO